theme/lumo/vaadin-custom-field-styles.html

```diff
@@ -43,20 +43,27 @@
         height: calc(0.4em - var(--lumo-space-xs));
       }
 
-      :host([focused]:not([disabled])) [part="label"] {
+      :host([focused]:not([readonly]):not([disabled])) [part="label"] {
```

Comment:
In vaadin-text-field styles these selectors are otherwise identical, but there they don't have `:not([disabled])`, since text field also sets `pointer-events: none` on the host when `[disabled]`, but we decided with Tomi and Jouni that it's probably safer for now to not disable pointer events from custom field. It can be done later if it seems to make sense.